src/sass/_forms.scss

```diff
@@ -482,3 +501,53 @@
     margin-right: $sph--x-small;
   }
 }
+
+.network-select-dropdown {
```

Comment:
This should probably live in a dedicated file for the network selector, similar to `_storage_pool_selector.scss`.